src/Mod/Path/PathScripts/PathDressupLeadInOut.py

```diff
@@ -653,35 +823,53 @@ def Activated(self):
         # check that the selection contains exactly what we want
         selection = FreeCADGui.Selection.getSelection()
         if len(selection) != 1:
-            PathLog.error(translate("Please select one path object")+"\n")
+            PathLog.error(
+                translate("Path_DressupLeadInOut", "Please select one path object")
+                + "\n"
+            )
             return
         baseObject = selection[0]
         if not baseObject.isDerivedFrom("Path::Feature"):
-            PathLog.error(translate("The selected object is not a path")+"\n")
+            PathLog.error(
+                translate("Path_DressupLeadInOut", "The selected object is not a path")
+                + "\n"
+            )
             return
         if baseObject.isDerivedFrom("Path::FeatureCompoundPython"):
-            PathLog.error(translate("Please select a Profile object"))
+            PathLog.error(
+                translate("Path_DressupLeadInOut", "Please select a Profile object")
+            )
             return
 
         # everything ok!
-        FreeCAD.ActiveDocument.openTransaction(translate("Create LeadInOut Dressup"))
+        FreeCAD.ActiveDocument.openTransaction(
+            translate("Path_DressupLeadInOut", "Create LeadInOut Dressup")
+        )
```

Comment:
@sliptonic 
Did CHennes state that `openTransaction()` calls did not need to be translated?  Just thinking I read that piece of information in one of his posts.  The messages for `openTransaction()` are nothing more than an internal log message, correct?  Or do they get displayed to the user?

Reply:
It was not me who said it, but that is my understanding of the situation.

Reply:
Thanks for the clarification. This is the reference I'm using for this work https://github.com/sliptonic/FreeCAD/wiki/Translation-Cleanup-Project#check-all-uses-of-translate @sliptonic, could you check this?

Reply:
You are correct.  That is on me.  transactions are not to be translated.  I will revise my coordination wiki page.